styles/amplenote/catppuccin.user.css

```diff
@@ -0,0 +1,237 @@
+      --color-background-nav-menu: @mantle;
+      --color-background-nav-menu-divider: @crust;
+      --color-background-popup-small: @surface0;
+      --color-background-reversed: @surface0;
+
+      --color-background-element-high-contrast: fade(@overlay2, 40%);
+      --color-background-element-medium-contrast: fade(@accent-color, 40%);
+      --color-background-element-low-contrast: @crust;
+
+      --color-background-note-menu-button: @surface0;
+      --color-background-banner: @surface0;
+      --color-background-nav-divider: @surface0;
+
+      --color-background-action-low-contrast: fade(@accent-color, 50%);
+      --color-background-action-medium-contrast: fade(@accent-color, 60%);
+      --color-background-action-high-contrast: @accent-color;
+
+      --color-border-primary: @surface0;
+      --color-border-tertiary: @surface1;
+
+      --color-text-reversed: @text;
+
+      --color-text-high-contrast: @text;
+      --color-text-nav-high-contrast: @text;
+      --color-text-medium-contrast: @subtext1;
+      --color-text-nav-medium-contrast: @subtext1;
+      --color-text-low-contrast: @subtext0;
+      --color-text-nav-low-contrast: @subtext0;
+      --color-text-lower-contrast: @overlay2;
+      --color-text-nav-lower-contrast: @surface2;
+
+      --color-text-error-low-contrast: @peach;
+      --color-text-error-medium-contrast: @red;
+      --color-text-error-high-contrast: @red;
+
+      --color-text-status: @green;
+      --color-text-link: @accent-color;
+      --color-text-button: @crust;
+
+      --color-background-task-value: @surface1;
+      --color-background-sidebar-task: @surface0;
+
+      --color-overlay-low-contrast: @accent-color;
+
+      --mdc-theme-surface: @base;
+    }
+
+    header.note-header-container .logo-wrapper .ample-svg-logo .logo-icon {
+      fill: @text;
+    }
+
+    .notes-list .notes-list-item:hover {
+      background-color: @crust;
+    }
+
+    .ample-editor .editor-tabs .tab:not(.current) {
+      border-bottom-color: @surface0 !important;
+    }
+
+    .referencing-notes,
+    .tab.current,
+    .editor-title-input-wrapper textarea,
+    .completed-tasks,
+    .no-tasks-message {
+      background-color: @base !important;
+    }
+
+    .new-note-button {
+      color: @crust !important;
+    }
+
+    .ample-editor .check-list-item.value-1 .row-wrapper {
+      border-left-color: @sapphire;
+    }
+
+    .menu-button.synced {
+      color: @green !important;
+    }
+
+    .primary-side-nav .section-link.default .shortcut-icon {
+      color: @yellow;
+    }
+
+    .ample-editor:not(.mobile-embed)
+      .check-list-item:hover:not(.expanded)
+      .row-wrapper {
+      background-color: @surface0;
+    }
+
+    .calendar-pane .fc .fc-timegrid-now-indicator-line {
+      border-color: @green;
+    }
+
+    .calendar-pane .fc .fc-timegrid .all-day-slot,
+    .calendar-pane .fc .fc-timegrid .fc-daygrid-day-events {
+      background-color: @base;
+    }
```

Comment:
The linter is complaining because this section can/should be folded into the new `.calender-pane .fc` section we just added below.